x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -145,6 +145,7 @@ const EventDetailsComponent: React.FC<Props> = ({
                 />
                 {enrichmentCount > 0 && (
                   <ThreatSummaryView
+                    data={data}
```

Comment:
Can you please explain why this component needs this new prop?

Reply:
I believe that a `data` prop is required for the `ActionCell` component to display the timeline icon as a part of the button row (similar to AlertSummary implementation [1](https://github.com/elastic/kibana/blob/745db3063a3763f6b704a0bf48d1f3093dba6161/x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx#L98-L103),  [2](https://github.com/elastic/kibana/blob/745db3063a3763f6b704a0bf48d1f3093dba6161/x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx#L131),  [3](https://github.com/elastic/kibana/blob/745db3063a3763f6b704a0bf48d1f3093dba6161/x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx#L160), [4](https://github.com/elastic/kibana/blob/745db3063a3763f6b704a0bf48d1f3093dba6161/x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.tsx#L181))

Reply:
See my above comment about props; while it's clear that `ActionCell` currently requires the same type as an element of `data` that we have here, it's not clear _why_ it needs those values on top of what we already have/provide to the draggable component.